src/components/Article/BottomMenu/DocPad/DocPadEditor/Toolbar/index.tsx

```
@@ -0,0 +1,243 @@
+
+type ToolbarProps = {
+  editor: Editor | null
+}
+
+const Toolbar: React.FC<ToolbarProps> = ({ editor }) => {
+  if (!editor) return null
+
+  const iconStyles = (tagOrObj?: string | object, obj?: object) => {
+    const isActive =
+      typeof tagOrObj === 'string'
+        ? editor && editor.isActive(tagOrObj, obj)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)